source/guides/decision-making-techniques.html.md

```diff
@@ -0,0 +1,62 @@
+
+This exercise is a very useful technique when making decisions on scope of a project or task. It can help the team focus on the key deliverables and improve the pace of delivery.
+
+You’ll first need to identify all potential tasks you could undertake, then sort them into the following categories:
+
+- must
+- should
+- could
+- won’t
+
+You must also provide a rationale as to why this choice was made, and ensure this is documented for future reference. 
+
+This exercise can be done as a group discussion, or as an individual exercise where team members asynchronously contribute their decisions. You can see an example of the latter [here when we were deciding on which guidance to ship as part of Exit this page](https://docs.google.com/spreadsheets/d/1_Cy7oVvzbGZhao9aZwEViaWG5-G4NcBMyqWT_cI-1NE/edit?gid=0#gid=0). This technique does however require an accountable decision maker, who will review the thoughts and rationale shared by the team and decide on how to proceed. 
+
+You can [read more about the MoSCoW framework here on ProductPlan](https://www.productplan.com/glossary/moscow-prioritization/)
+
+There are also fun variations of this exercise such as ‘Cupcake — Cake — Wedding Cake’ as seen [here on Fun Retrospectives](https://www.funretrospectives.com/cupcake-cake-wedding-cake/).
+
+### Risk vs benefit analysis / pros vs cons
+
+A fairly simple exercise which can be used to make a decision on a particular topic. It allows the team to list all the potential benefits associated with doing a thing, and then all the potential risks. The team can then discuss how the risks can be mitigated. 
+
+This exercise can also highlight risks which are too great. These are usually the risks where it’s difficult to mitigate.
+
+This exercise is useful for building confidence within a team when making a decision. It ensures all aspects have been explored, and there is a plan for what to do if a risk materialises.
+
+## Recording decisions
+
+As well as making decisions, it’s also important to record key decisions which are made as part of a project. This ensures the rationale can be revisited in the future, by any team member. This is particularly useful when providing support to users who want more information about our decision making.
+
+There are a few tools you can use to record decisions:
+
+- a decision log, such as the one used [here when developing Navigation components](https://docs.google.com/spreadsheets/d/1zduTR-kZc5j28PFMi98-KNoP8W8S-8iJtisf9nLAbIM/edit?gid=1037439525#gid=1037439525) 
+- an update to the [community backlog](https://github.com/orgs/alphagov/projects/43/views/2) entry for a particular style, component or pattern
+- a comment on the relevant GitHub issue
+- a GitHub repository such as [govuk-design-system-architecture here](https://github.com/alphagov/govuk-design-system-architecture)
+- a design history such as [the ones created here by the Department for Education](​https://design-histories.education.gov.uk/)
```

Comment:
Link doesn't work. You could reference https://x-govuk.github.io/govuk-design-history/ if you can't find the link.

Reply:
Yes this is a better link. And for some reason that original URL won't link out